LongWarOfTheChosen/Src/LW_Overhaul/Classes/UIMission_LWCustomMission.uc

```diff
@@ -175,21 +180,26 @@ simulated function String GetMissionTitle()
 
 simulated function BuildScreen()
 {
-	// Add Interception warning and Shadow Chamber info 
-	BindLibraryItem();
-	`LWACTIVITYMGR.UpdateMissionData(GetMission());
-	BuildMissionPanel();
-	BuildOptionsPanel();
-	class'UIUtilities_LW'.static.BuildMissionInfoPanel(self, MissionRef);
-
 	PlaySFX(GetSFX());
-
 	XComHQPresentationLayer(Movie.Pres).CAMSaveCurrentLocation();
+	`HQPres.StrategyMap2D.HideCursor();
 
 	if(bInstantInterp)
 		XComHQPresentationLayer(Movie.Pres).CAMLookAtEarth(GetMission().Get2DLocation(), CAMERA_ZOOM, 0);
 	else
 		XComHQPresentationLayer(Movie.Pres).CAMLookAtEarth(GetMission().Get2DLocation(), CAMERA_ZOOM);
+
+	// Add Interception warning and Shadow Chamber info
+	`LWACTIVITYMGR.UpdateMissionData(GetMission());
+
+	// Base version is responsible for showing most mission info, including the mission and options panels,
+	// shadow chamber, chosen, sitreps, etc.
+	super.BuildScreen();
+
+	// Add the LW-specific mission info: squad size, concealment type, evac mode, etc.
+	class'UIUtilities_LW'.static.BuildMissionInfoPanel(self, MissionRef);
+
+	BuildConfirmPanel();
```

Comment:
Is this worth it? I'm pretty sure it doesn't do anything. I think it's confusing to call a method that looks like it should do something with the Confirm and Ignore buttons, but in fact does nothing. Have I missed something?